### PR TITLE
feat(sdk): decouple the status updating logic from the original updating logic.

### PR DESCRIPTION
**Pull request by jackworkgit**
1. Add a new endpoint to update the status of the org and process the related connection.
2. Decouple the status updating logic from updating the display name.
3. Add default metadata when creating org.

### CODE REVIEW

**Review by katykk193 — APPROVED** [review]

**Comment by DaveXiong** [comment]
Hey @jackworkgit, Pls help to add unit test cases so that can pass the sonarcloud check.

**Review by pori — APPROVED** [review]

File: vortex-java-sdk/vortex-java-sdk-iam/src/main/java/com/consoleconnect/vortex/iam/controller/MgmtOrganizationController.java [hunk 1]
```diff
@@ -154,4 +155,14 @@ public Mono<HttpResponse<OrganizationConnection>> updateConnection(
         HttpResponse.ok(
             service.updateConnection(orgId, updateConnectionDto, authenticationToken.getName())));
```
Comment (DaveXiong) [inline vortex-java-sdk/vortex-java-sdk-iam/src/main/java/com/consoleconnect/vortex/iam/controller/MgmtOrganizationController.java:156]: Pls remove `status` from UpdateOrganizationDto.java

File: vortex-java-sdk/vortex-java-sdk-iam/src/test/resources/application.yaml [hunk 2]
```diff
@@ -37,11 +38,30 @@ spring:
...
-    max-in-memory-size: 10MB
+    enabled: false
+  security:
+    oauth2:
+      resourceserver:
+        jwt:
+          issuer-uri: ${app.iam.jwt.issuer-uri}
+          jwk-set-uri: ${app.iam.jwt.jwk-set-uri}
+          audiences: ${app.iam.jwt.audience}
+  cloud:
```
Comment (DaveXiong) [inline vortex-java-sdk/vortex-java-sdk-iam/src/test/resources/application.yaml:50]: pls delete `cloud.gateway` configuration, useless in this module

File: vortex-java-sdk/vortex-java-sdk-iam/src/test/resources/application.yaml [hunk 3]
```diff
@@ -74,38 +94,71 @@ springdoc:
   servers:
     - url: http://localhost:8000
       description: localhost
-
 logging:
   level:
     com.consoleconnect.vortex: DEBUG
+    org.springframework.security: DEBUG
+    org.springframework.cloud.gateway: DEBUG
```
Comment (DaveXiong) [inline vortex-java-sdk/vortex-java-sdk-iam/src/test/resources/application.yaml:101]: delete `org.springframework.cloud.gateway: DEBUG`

File: vortex-java-sdk/vortex-java-sdk-iam/src/test/resources/application.yaml [hunk 4]
```diff
@@ -74,38 +94,71 @@ springdoc:
...
+            - DELETE
+          roles:
+            - ORG_ADMIN
+        - path: /organization/**
+          http-methods:
+            - GET
+          roles:
+            - ORG_ADMIN
+            - ORG_MEMBER
   gateway:
```
Comment (DaveXiong) [inline vortex-java-sdk/vortex-java-sdk-iam/src/test/resources/application.yaml:160]: delete 'gateway`

**Review by DaveXiong — COMMENTED** [review]
comments added

File: vortex-java-sdk/vortex-java-sdk-iam/src/test/resources/application.yaml (hunk 4 above)
Comment (jackworkgit) [reply vortex-java-sdk/vortex-java-sdk-iam/src/test/resources/application.yaml:160]: ok

File: vortex-java-sdk/vortex-java-sdk-iam/src/test/resources/application.yaml (hunk 2 above)
Comment (jackworkgit) [reply vortex-java-sdk/vortex-java-sdk-iam/src/test/resources/application.yaml:50]: ok

File: vortex-java-sdk/vortex-java-sdk-iam/src/test/resources/application.yaml (hunk 3 above)
Comment (jackworkgit) [reply vortex-java-sdk/vortex-java-sdk-iam/src/test/resources/application.yaml]: ok

File: vortex-java-sdk/vortex-java-sdk-iam/src/main/java/com/consoleconnect/vortex/iam/controller/MgmtOrganizationController.java (hunk 1 above)
Comment (jackworkgit) [reply vortex-java-sdk/vortex-java-sdk-iam/src/main/java/com/consoleconnect/vortex/iam/controller/MgmtOrganizationController.java:156]: ok

**Review by DaveXiong — APPROVED** [review]
LGTM

**Pull request merged**